distribution/docs/src/main/resources/content/_configuring/reusing-configurations.adoc

```diff
@@ -42,6 +46,6 @@ This file represents the decrypted version of the `.dar` file.
 ** importing from a system installed on a different OS
 ** importing from a system installed in a different directory location
 * To keep the export/import process simple and consistent, all system configuration files are required to be under the `${home_directory}` directory and not be softlinks. Presence of external files or symbolic links during export will not fail the export; they will yield warnings. It will be up to the adminsitrator to manually copy these files over to the new system before proceeding with the import. The import process will verify their presence and consistency and yield warnings if they don't match the original files.
```

Comment:
out of scope, but there is a typo on this line : `adminsitrator`

Reply:
Still in scope. Thanks